specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/network/rfc7540/connection.management/server.rst.stream.last.frame/client.rpt

```diff
@@ -0,0 +1,105 @@
+#
```

Comment:
Why is this scenario called `server.rst.stream.last.frame`? I didn't get what the `last.frame` part was about.

Also, this feels like a flow control overflow scenario, not a connection management scenario, agree?

Reply:
Yeah, agree it is about sending data before receiving WINDOWS_UPDATE.

Reply:
Not sure as well, I think we can say `overflow `